login/LoginRequest.js

```diff
@@ -0,0 +1,7 @@
+class LoginRequest{
+    constructor(login,password){
+        this.login=login;
+        this.password=password;
+    }
+};
+module.exports = LoginRequest;
```

Comment:
Are you sure this is the modern way to share your classes in node.js? I've found an article https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/ where the author uses things like "export class LoginRequest" and "export default class LoginRequest" to export his classes and later does something like import { LoginRequest } from "./login-request.js"; in the files he wants to access LoginRequest in, I think it looks way clearer and saves you a line of code in every class.

Reply:
ok,i will try to fix it